### PR TITLE
docs: replace word therefore, plus other small changes

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Replace word `therefore` with `so`
- Rewrite sentences to drop `therefore` or `so`
- Some other small changes

## Context

The plain language guidelines recommends replacing `therefore` with `so`.

Not closing any issue with this PR.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/getting-started/private-packages.md [hunk 1]
```diff
@@ -371,8 +371,8 @@ The solution to this is that you should break your presets into public and priva
 
 ### Encrypting secrets
 
-It is strongly recommended that you don't commit secrets to repositories, including private ones, and this includes secrets needed by Renovate to access private modules.
-Therefore the preferred approach to secrets is that the bot administrator configures them as `hostRules` which are then applied to all repositories which the bot accesses.
+It is strongly recommended that you avoid committing secrets to repositories, including private ones, and this includes secrets needed by Renovate to access private modules.
```
Comment (HonkingGoose) [inline docs/usage/getting-started/private-packages.md:374]: We should put our point positively: [^quote]

> ## Use positive language
> 
> We’re accustomed to thinking and speaking positively. When we write in the negative, we place another stumbling block in audience’s way and make it more difficult for them to understand us. When you’re going to meet a friend at the airport, do you say, “If you fail to arrive by 5:00, I cannot pick you up,” or do you say, “You have to arrive by 5:00 if you want me to pick you up?”

[^quote]: [Plain language guidelines, use positive language](https://www.plainlanguage.gov/guidelines/concise/use-positive-language/)

File: lib/modules/datasource/docker/readme.md [hunk 2]
```diff
@@ -2,4 +2,4 @@ This datasource identifies an image's source repository according to the [pre-de
 
 This datasource looks for the metadata of the **latest stable** image found on the Docker registry and uses the value of the label `org.opencontainers.image.source` and `org.label-schema.vcs-url` as the `sourceUrl`.
 
-The [Label Schema](https://label-schema.org/) is superseded by OCI annotations, therefore `org.opencontainers.image.source` label should be preferred.
+The [Label Schema](https://label-schema.org/) is superseded by OCI annotations, so the `org.opencontainers.image.source` label should be preferred.
```
Comment (HonkingGoose) [inline lib/modules/datasource/docker/readme.md:5]: I'm not happy with "should be preferred"  as it doesn't tell me who or what prefers something in this sentence.

Do we recommend that our readers use OCI annotations and the `org.opencontainers.image.source` label?

Comment (rarkins) [reply lib/modules/datasource/docker/readme.md:5]: "use the .... if possible" ?

**Review by rarkins — COMMENTED** [review]
Are you sure about "So" at the start of sentences? It doesn't sit right for me.

File: lib/modules/datasource/docker/readme.md (hunk 2 above)
Comment (HonkingGoose) [reply lib/modules/datasource/docker/readme.md:5]:
```suggestion
The [Label Schema](https://label-schema.org/) is superseded by OCI annotations, use the `org.opencontainers.image.source` label if possible.
```

**Comment by HonkingGoose** [comment]
> Are you sure about "So" at the start of sentences? It doesn't sit right for me.

I think the main problem is that the sentences are clunky. Just replacing `therefore` with `so` doesn't fix the clunky sentences.

I'll try to rewrite the sentences so we don't need `so` or `therefore` at the beginning.

**Review by HonkingGoose — COMMENTED** [review]
Let's see how you feel about these suggestions. 😄

**Pull request merged**